sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 * Added method `get_topic_sender` in `ServiceBusClient` to get a `ServiceBusSender` for a topic.
 * Added method `get_subscription_receiver` in `ServiceBusClient` to get a `ServiceBusReceiver` for a subscription under specific topic.
+* Added support for scheduling messages and scheduled message cancellation.
```

Comment:
Is it worth noting scheduled_enqueue_time_utc here?  or overkill you think?  I'm ambivalent.

Reply:
I think that's some detail user can find out through reading docstring/api reference.